t/run_tests

```diff
@@ -3,14 +3,25 @@ set -eu
 
 cd $(dirname $0)
 
+# Put git_filter_repo.py on the front of PYTHONPATH
+export PYTHONPATH=$PWD/..${PYTHONPATH:+:$PYTHONPATH}
```

Comment:
$PWD should be quoted.

Reply:
Indeed.

Reply:
Fixed in fa566a6421196fdac07765a695674fdf5837df51.

Reply:
Awesome, thanks!